packages/manager/modules/pci/src/projects/project/private-networks/private-networks.component.js

```diff
@@ -10,6 +10,7 @@ export default {
     projectId: '<',
     networkId: '<',
     onListParamChange: '<',
+    canDisplayPciAnnouncementBanner: '<',
```

Comment:
i think adding a "canDisplayPciAnnouncementBanner" in every component binding isn't practical as it makes components less reusable (more coupling)

can you find another way to check if the feature is available ? maybe in the controller of your pci-annoucement-banner component or using a service

Reply:
True, done.